fabtests/common/shared.c

```diff
@@ -664,12 +674,20 @@ int ft_alloc_active_res(struct fi_info *fi)
 	return 0;
 }
 
-static void ft_init(void)
+static int ft_init(void)
 {
 	tx_seq = 0;
 	rx_seq = 0;
 	tx_cq_cntr = 0;
 	rx_cq_cntr = 0;
+
+	//If using device memory for transfers, require OOB address
+	//exchange because extra steps are involved when passing
+	//device buffers into fi_av_insert
+	if (opts.options & FT_OPT_USE_DEVICE)
+		opts.options |= FT_OPT_OOB_ADDR_EXCH;
```

Comment:
Not quite sure I follow. Is this to avoid passing a device buffer into fi_av_insert?

Reply:
Yes, this change was made in response to my comments.  The previous patch had code to copy the address data that was received into a message into a host bounce buffer for AV handling.  For someone looking at the code, the need to use a bounce buffer isn't obvious.  This change removes the bounce buffer and reverts the AV code back to the current code.